src/identity/components/GlobalHeader/GlobalHeaderDropdown/GlobalHeaderTypeAheadMenu.tsx

```diff
@@ -25,15 +34,15 @@ export class GlobalHeaderTypeAheadMenu extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props)
     this.state = {
-      searchTerm: '',
       queryResults: this.props.typeAheadMenuOptions,
+      searchTerm: '',
       selectedItem: this.props.defaultSelectedItem,
     }
   }
 
-  private selectAllTextInInput = (event?: ChangeEvent<HTMLInputElement>) => {
-    if (event) {
-      event.target.select()
+  private selectAllTextInInput = (e?: ChangeEvent<HTMLInputElement>) => {
+    if (e) {
+      e.target.select()
```

Comment:
why the change from `event` to `e`?

I'm not a fan of using `e` for events, since in JavaScript, `e` can be `event` or `error` and is often used interchangeably. Try searching the codebase for just `e` and finding an event handler or an error handler. (Regex searching for `\(e` yielded 1896 matches across 585 files for me). There's almost no benefit to using `e` over `event`, other than it's slightly faster to type, and it's faster to read, at the expense of taking more work to comprehend and map to a value when holding this block of code in your head.

Reply:
Agreed - but in this file, we're using `event` as a variable, so my thinking was that it's confusing to have a variable (even one in another scope) that is called `event`.